dependencies/bullseye/hammer_cli_foreman_host_reports/control

```diff
@@ -0,0 +1,25 @@
+Source: ruby-hammer-cli-foreman-host-reports
+Section: ruby
+Priority: optional
+Maintainer: Debian Ruby Extras Maintainers <pkg-ruby-extras-maintainers@lists.alioth.debian.org>
+Uploaders: <>
```

Comment:
this isn't true, as this package is not in debian, please set to yourself, or some generic foreman address.